controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/PathGroup.java

```diff
@@ -20,7 +20,8 @@
 enum PathGroup {
 
     /** Paths used for system management by operators. */
-    operator("/controller/v1/{*}",
+    operator("/configserver/v1/{*}",
```

Comment:
I still don't understand this... Now `/configserver/v1` is on the same list as `/zone/v2`, but `/zone/v2` read is available for everyone, yet if I don't have `/configserver/v1` in here, all tests in `ConfigServerApiHandlerTest` fail with 403 (including GETs).

If this is enough to lock down read & write to operators, why does `/zone/v2` read work?

Reply:
Figured it out